---
layout: post
title: Poc ID via mtORF POC RAPID Plate 001
date: '2024-04-12'
categories: Sanger
tags: [DNA, Gel, PCR, Pocillopora, mtORF]
---
# Pocillopora species ID COTS POC RAPID

## Equipment and Reagents

- Sample Preservation and Lysis [Zymo Research DNA/RNA Shield R1100-250](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/Zymo_r1100-250_dna_rna_shield.pdf)
- DNA Extraction 96 Well Plate [Zymo Research Quick-DNA™ 96 D3010](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/_d3010_d3011_d3012_quick-dna_96_kit.pdf) 
- DNA Extraction Individual Samples [Zymo Research Quick-DNA™ Miniprep plus kit extraction Cat D4068](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/d4068_d4069_quick-dna_miniprep_plus_kit.pdf) 
- Proteinase K [Zymo Proteinase K with Storage BufferZymo D3001-2-20](https://www.zymoresearch.com/products/proteinase-k-w-storage-buffer-set)
- PK Digestion Buffer [Zymo PK Digestion Buffer R1200-1-20](https://www.zymoresearch.com/products/pk-digestion-buffer)   
- Foward primer [FatP6.1 200µM Stock IDT](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/Fatp6.1_IDT_Spec_328104852.pdf) 
- Reverse primer [RORF 200µM Stock IDT](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/RORF_IDT_Spec_328104853.pdf)         
- Master Mix [EmeraldAmp GT PCR Master Mix RR320A](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/TaKaRa_Emerald_RR320A_DS.pdf)
- EmeraldAmp GT PCR Master Mix is a loading-dye-added version of EmeraldAmp MAX PCR Master Mix that is optimized for great performance and convenience in both standard and high-throughput PCR applications.
- Loading Dye [NEB 6X Purple Loading Dye NEB Cat # B7024S](https://www.neb.com/en-us/products/b7025-gel-loading-dye-purple-6x-no-sds)        
- Gel Stain [Biotium GelGreen Nucleic Acid Gel Stain, 10,000X in Water Fisher Cat NC9728313](https://www.fishersci.com/shop/products/gel-green-stain-5ml/NC9728313#?keyword=NC9728313)
- DNA Ladder Image NEB Cat N0550G [1kb Gel ladder NEB Cat N0550G](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/NEB_1kbplus_purple_N0550G.gif?raw=true)
- DNA Ladder NEB Cat N0550G [1kb Gel ladder NEB Cat N0550G](https://www.neb.com/en-us/products/n0550-quick-load-purple-1-kb-plus-dna-ladder)
- KapaPure Beads Fisher 50-196-5220 [Roche Diagnostics 07983280001](https://www.fishersci.com/shop/products/kapa-pure-beads-4/501965220) 
- Eppendorf Deepwell Plate 96/2000 µL plates [Eppendorf Cat # 951033502](https://www.eppendorf.com/us-en/eShop-Products/Laboratory-Consumables/Plates/Eppendorf-Deepwell-Plates-p-951033502)

 


# 11 April 2024
## DNA Extraction  

DNA was extracted with[Zymo Research Quick-DNA™ 96 Kit D3010](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/_d3010_d3011_d3012_quick-dna_96_kit.pdf) 

Used Page 10 [Proteinase K Digestion with DNA/RNA Shield](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240411_ZymoQuickDNA96_notes.jpg?raw=true)

Used [Eppendorf Deepwell Plate 96/2000 µL plates Catalog No. 951033502](https://www.eppendorf.com/us-en/eShop-Products/Laboratory-Consumables/Plates/Eppendorf-Deepwell-Plates-p-951033502) for reagent and sample mixing prior to adding to Zymo column plate

Need Proteinase K with Storage Buffer (Zymo D3001-2-20) and PK Digestion Buffer (Zymo R1200-1-20)

For each of 3 tubes of Proteinase K with Storage Buffer (Zymo D3001-2-20) I added 1040µl of Proteinas K Storage Buffer to lyophilized Proteinase K 20mg tube and mixed by vortex.



- First 150µl of new DNA/RNA Shield was added to each well in the Eppendorf 96 Deepwell plate

- Next 30µl of PK Digestion buffer and 15µl of resusdended Proteinase K solution was added to each well.

- Next 150µl of sample added to each well and samples were mixed with solution.

The samples were extracted according to the manufacturer’s instructions for the Quick DNA 96 Kit for samples stored in DNA/RNA Shield including the addition of 15µl of Proteinase K (20mg ml-1). 


Zymo Kit Genomic Lysis Buffer was prepared according to [Instructions Page 4](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/_d3010_d3011_d3012_quick-dna_96_kit.pdf). 500µl of beta-mercaptoethanol was added to the 100ml bottle of Genomic Lysis Buffer. 


- Next 4 volumes of Zymo Kit Genomic Lysis Buffer was added for each volume of the sample digestion (e.g., add 1200µl of Genomic Lysis Buffer to 300µl of sample digestion). Samples were mixed via pipetting and the mixutres were incubated for 5 minutes at room temperature. 

_MASSIVE PROBLEM HERE IN ZYMO INSTRUCTIONS FOR THE KIT INSTRUCTIONS PROVIDED. THERE IS ONLY 100ML OF GENOMIC LYSIS BUFFER PROVIDED, BUT THE KIT INSTRUCTIONS DO NOT TELL YOU TO ORDER MORE PRIOR TO STARTING THE KIT. THEREFORE WE WERE LEFT WITH NO GENOMIC LYSIS BUFFER FOR COLUMNS 11 AND 12 OF PLATE 001. THEREFORE WE MOVED FORWARD WITH EXTRACTIONS OF 80 SAMPLES FROM PLATE 001 COLUMNS 1-10 ALL ROWS. COLUMNS 11 AND 12 WERE MOVED TO NEW INDIVIDUALLY LABELLED TUBES AND STORED IN THE -20°C FREEZER._

![Page 10 of kit protocol](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240411_ZymoQuickDNA96_notes.jpg?raw=true)

Each well of the Silicon-A plate set on the collection plate can hold 600µl, but this brings the liquid too close to the top and increases chances of carry over to other wells. Becasue of this, I added the sample in 4 steps: 
1) 400µl and then 5 minute spin at 1000 rcf
1) 400µl and then 5 minute spin at 1000 rcf
1) 400µl and then 5 minute spin at 2500 rcf
1) 400µl and then 5 minute spin at 2500 rcf

The waste was removed from the collection plate after each centrifugation step. 

- Next 200µl of DNA Pre-wash buffer was added to each well and then the plate was spun for 5 minute at 2500 rcf

- Next 300µl of g-DNA wash buffer was added to each well and then the plate was spun for 5 minute at 2500 rcf

- DNA was then eluted in 60µl of kit Elution Buffer warmed to 70°C. DNA extractions were completed on 11 April 2024 and DNA stored at -20°C in the elution plate covered with a foil seal.

#### Samples

![Pocillopora gDNA extraction Samples: Plate 001](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240411_POC_COTS_gDNA_Plate001.png?raw=true)



# 12 April 2024
## Quantification of gDNA   
Used Lane lab nanodrop and kit elution buffer as blank to quantify the first column of DNA on the plate. 

Sample ID | Plate | Well |Project | Collection Date |  ng/µl | A260/280 | A260/230|
---|---|---| ---|---|---|---|---|
POC-466 | Plate 001 | A1 |POC HISTOLOGY | Sept 2022 |  5.1|2.14|0.97|
POC-478 | Plate 001 | B1 |POC HISTOLOGY | Sept 2022 |  7.6|2.16|2.04|
FLOW-005 | Plate 001 | C1 |FLOW SKELETON | March 2024 | 15.9|2.08|2.14|
FLOW-020 | Plate 001 | D1 |FLOW SKELETON | March 2024 |10.6|1.96|1.77|
FLOW-033 | Plate 001 | E1 |FLOW SKELETON | March 2024 |  6.8|2.40|0.85|
TPC REC-001 | Plate 001 | F1 |TPC COTS RAPID | March 2024 |  12.3|2.00|1.89|
TPC REC-013 | Plate 001 | G1 |TPC COTS RAPID | March 2024 |  10.3|2.04|1.19|
TPC REC-025 | Plate 001 | H1 |TPC COTS RAPID | March 2024 |  22.8|1.98|1.58|
Elution Buffer | NA | NA |NA | NA |  0.1|-0.46|0.36|


## PCR
### Prepare Reagents
Primers - [Flot et al. 2008](https://www.sciencedirect.com/science/article/pii/S1631069107003812?via%3Dihub)   
- Forward primer FatP6.1	(Added 50µl of stock 2mg/ml plus 950µl of Nuclease Free water and made 3 aliquots of ~320µl each and stored in reagent box at -20)   
- FatP6.1 (5′-TTTGGGSATTCGTTTAGCAG-3′)    
- Reverse primer RORF	(Added 50µl of stock 2mg/ml plus 950µl of Nuclease Free water and made 3 aliquots of ~320µl each and stored in reagent box at -20)     
- RORF (5′-SCCAATATGTTAAACASCATGTCA-3′)    
- Master Mix [EmeraldAmp GT PCR Master Mix]()
- ```EmeraldAmp GT PCR Master Mix is a loading-dye-added version of EmeraldAmp MAX PCR Master Mix that is optimized for great performance and convenience in both standard and high-throughput PCR applications.```
**EmeraldAmp GT PCR Master Mix was recieved warm on arrival and stored at -20°C**

### Master Mix: For 82 rxns +3 = 85 

Reagent | 1Rxn µl | 85 Rxn µl |  
---|---|---|
EmeraldAmp GT Mix (2x)| 		12.5	|	1062.5|
F primer FatP6.1 (10µM) |	0.3	|	25.5
R primer RORF		(10µM)  |	0.3	|	25.5
DNA		|				1		| NA	
H2O			|			10.9	|	926.5
Total volume 		|	25		|	----

24µl added to each of 82 wells. The total master mix was not enough and 2 more wells needed to be generated individually (H8 and H9). Next time add more for pipette loss.

![Pocillopora mtORF amplification samples: Plate 001](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240412_POC_COTS_mtORF_PCR_Plate001.png?raw=true)

### Thermal Cycling Conditions 
- [94°C 60 secondes] 1 cycle
- [94°C 30 sec,53°C 30 sec, 72°C 75 sec] 30 cycles
- [72°C 5 minutes] 1 cycle
- [4°C infinity]

### PCR Gel protocol
DNA was assessed with a 1.5% agarose gel in 1x TAE buffer (2.25g agarose in 150ml of 1X TAE, Tris base, acetic acid and EDTA) run at for 40 mins at 66 V and stained with Biotium GelGreen Nucleic Acid Gel Stain (10,000X in Water Fisher Cat NC9728313). 

### Gel Results
   
![Pocillopora mtORF PCR and Pocillopora gDNA 4/12/24](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240412_mtORF_gDNA_gel_Pocillopora.png?raw=true)

[Pocillopora mtORF PCR and Pocillopora gDNA Gel results 4/12/24](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240412_mtORF_gDNA_gel_Pocillopora.png?raw=true)

Bands of ~1000 bp are present in all samples except the negative control for the mtORF amplicons.

Bands of HMW were present for all of the gDNA. 

Amplified samples were saved in PCR plate for cleanup for mtORF For primer Sanger sequencing in 4°C fridge by PCR bench.


# 20240419

## DNA extraction of remaining 16 samples from Plate001

DNA was extracted with[Zymo Research Quick-DNA™ Miniprep plus kit extraction Cat D4068](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/d4068_d4069_quick-dna_miniprep_plus_kit.pdf) 

ProteinaseK was already added from steps above, so started at step 3 on page 7.

- Added 300µl of Genomic Binding Buffer to the sample and mixed with the pipette.
- Transferred the mixture to a Zymo-Spin™ IIC-XLR Column in a
Collection Tube. Centrifuged at 12,000 x g for 1 minute. Discarded the
flow through and collection tube.
- Added 400 µl DNA Pre-Wash Buffer to the spin column in a new
Collection Tube. Centrifuged at 12,000 x g for 1 minute. Emptied the
Collection Tube. 
- Added 700 µl g-DNA Wash Buffer to the spin column. Centrifuged at
12,000 x g for 1 minute. Emptied the Collection Tube.
- Added 200 µl g-DNA Wash Buffer to the spin column. Centrifuged at
12,000 x g for 1 minute. Discarded the Collection Tube with the flow
through.
- TransferRED the spin column to a clean microcentrifuge tube. Add 60 µl
DNA Elution Buffer that was heated to 70°C and incubated for 5
minutes at room temperature, then centrifuged at 20,000 x g  for
1 minute to elute the DNA. 
- The eluted DNA was then quantified via nanodrop, used for PCR, and stored -20ºC in box 1 of a new vertical metal rack in chest freezer for future use.
- FLOW-003 lid broke in spin
- FLOW-043 lid cracked in spin


## Quantification of gDNA   
Used Lane lab nanodrop and kit elution buffer as blank to quantify the 16 samples.

Sample ID | Plate | Well |Project | Collection Date |  ng/µl | A260/280 | A260/230|
---|---|---| ---|---|---|---|---|
POC-476 | Plate 001 | A11 |POC HISTOLOGY | Sept 2022 |  14.8|2.10|0.63|
FLOW-003 | Plate 001 | C11 |FLOW SKELETON | March 2024 |  38.4|1.91|1.97|
FLOW-017 | Plate 001 | C11 |FLOW SKELETON | March 2024 |  28.7|2.01|0.11|
FLOW-031 | Plate 001 | D11 |FLOW SKELETON | March 2024 |42.9|1.97|1.96|
FLOW-043 | Plate 001 | E11 |FLOW SKELETON | March 2024 |  45.1|1.94|1.88|
TPC REC-011 | Plate 001 | F11 |TPC COTS RAPID | March 2024 | 11.1|2.20|0.04|
TPC REC-023 | Plate 001 | G11 |TPC COTS RAPID | March 2024 |  94.0|1.92|0.38|
TPC REC-035 | Plate 001 | H11 |TPC COTS RAPID | March 2024 |2.2|3.04|0.19|
POC-477 | Plate 001 | A11 |POC HISTOLOGY | Sept 2022 |  15.7|2.00|1.44|
FLOW-004 | Plate 001 | C11 |FLOW SKELETON | March 2024 | 55.3|1.93|1.93|
FLOW-019 | Plate 001 | C11 |FLOW SKELETON | March 2024 | 38.4|1.97|2.11|
FLOW-032 | Plate 001 | D11 |FLOW SKELETON | March 2024 |46.3|1.86|1.54|
FLOW-044 | Plate 001 | E11 |FLOW SKELETON | March 2024 |20.5|1.99|1.00|
TPC REC-012 | Plate 001 | F11 |TPC COTS RAPID | March 2024 | 15.0|2.10|1.58|
TPC REC-024 | Plate 001 | G11 |TPC COTS RAPID | March 2024 | 58.2|1.93|2.39|
TPC REC-036 | Plate 001 | H11 |TPC COTS RAPID | March 2024 | 6.8|2.66|0.84|
Elution Buffer| NA | H11 |TPC COTS RAPID | March 2024 | 0.5|-1.24|0.39|   


### Master Mix: For 18 rxns +2 = 20 

Reagent | 1Rxn µl | 20 Rxn µl |  
---|---|---|
EmeraldAmp GT Mix (2x)| 		12.5	|	250|
F primer FatP6.1 (10µM) |	0.3	|	6
R primer RORF		(10µM)  |	0.3	|	6
DNA		|				1		| NA	
H2O			|			10.9	|	218
Total volume 		|	25		|	----

24µl of master mix was added to each of 18 wells
positive control = POC 156 Spawning
negative control = master mix water

### Thermal Cycling Conditions 
- [94°C 60 secondes] 1 cycle
- [94°C 30 sec,53°C 30 sec, 72°C 75 sec] 30 cycles
- [72°C 5 minutes] 1 cycle
- [4°C infinity]

### PCR Gel protocol
DNA was assessed with a small 28 well 1.5% agarose gel in 1x TAE buffer (1.12g agarose in 75ml of 1X TAE, Tris base, acetic acid and EDTA) run at for 20 mins at 100 V and stained with 1.5 µl of Biotium GelGreen Nucleic Acid Gel Stain (10,000X in Water Fisher Cat NC9728313). 4 µl of sample or 4µl of NEB 1kb plus purple ladder were added to each well.

### Gel Results
   
![Pocillopora mtORF PCR 4/19/24](https://github.com/hputnam/Putnam_Lab_Notebook/blob/master/images/20240419_mtORF_gel_Pocillopora.jpg?raw=true)

PCR bands of the expected size were seen in all lanes except the negative control.
PCR product was stored in the 4°C fridge by the PCR bench with the rest of Plate 001.


# 202404XX

### PCR Cleanup


1X Bead Cleanup (post PCR bench)

**KAPAPure beads are light sensitive**

_Take KAPA pure beads out of 4 degree fridge ~30 minutes before use to get to room temperature, and swirl to even out the beads (which settle on the bottom), and keep them in a drawer while warming out of the light_

Make fresh 80% ethanol for each day. For example in 50mL conical add 40mL of 100% ethanol and 10ml of ultrapure water.

Take the PCR plates out of either fridge or thermocyler and spin down in centrifuge at 2500 for 1 minute

_You can either add beads 1 well at a time or by row using a multichannel and a liquid trough. If you have more than 16 samples, it saves time to use a trough._

Take beads out of the drawer and swirl again. If using a trough, calculate how many beads you'll need: 100ul * n and pipette that volume of beads into the trough

Add 20ul of beads to each well, slowly because the bead liquid is very viscus. Pipette up and down at least 10 times per sample to mix, pipette slowly to avoid bubbles forming. You have to completely mix gently here, your DNA is binding to the beads in this step.

Repeat for all rows/samples, avoid bubbles because they can effect magnetic binding, tap down plate gently if some bubbles appear.

Place your bead-sample plate to incubate on the rotating shaker for 15 minutes at 200 speed to facilitate DNA binding.

After incubation, place the plate on the magnet plate (white plate with silver metal knobs), and place back on the shaker for ~5 minutes. Larger volumes can take longer for the beads to magnetize so shaking helps.

Take plate back to bench. Set up a trough for "supernatant", a trough for EtOH, and for ultra pure water
Pour some EtOH and water into their troughs but leave the supernatant one empty
When the liquid in the wells has gone clear, carefully use the multichannel to pipette out the clear supernatant without disturbing the beads (set pipettes to ~175ul), it helps to slide the tips down the sides of the tubes without the beads. The clear sides alternates each row. 

Remove supernatant from every well and discard supernatant in the waste labeled trough and change tips. 


Add 200ul of 80% EtOH to each well carefully without disturbing the beads, again you can put the tips on the side of the wells without the beads
Remove the clear supernatant from each well without disturbing the beads and discard into the supernatant trough, changing tips for each well.

Repeat the 80% ethanol washes for a total of 3 times. 

While the plate is on the magnet use the p20 multichannel pipette to suck up any remaining ethanol at the bottom of the wells of the plate 

Wait ~2 minutes to let any residual ethanol to dry

**Do not overdry the beads**


Take plate off the magnet into a regular rack

Set multichannel to 25ul and add 25ul of ultrapure nuclease-free water to each well of the plate. 

Carefully add the water to the beads, you'll have to pipette up and down slowly as long as it takes to resuspend the beads off the side of the well. Alternate the tip position to get beads stuck on opposite sides of the wells. Avoid making bubbles and repeat for every well


Place plate on the shaker at 200 rpm for 5 minutes

Get another plate ready for the cleaned product

Once done shaking, place the plate back on the magnet and wait until the wells become clear

Remove the 25ul of clear supernatant from each well and into the new 96-well plate

Seal the plate with a labeled foil seal and place in the fridge if preparing for sequencing in the next 24h, or freezer if it will be stored 24h


### Sanger sequencing setup

Dilute For primer to 3.2 µM

use a multichannel to take 2ul from each sample into a new plate, and add 10ul of nuclease free water to each well for a 1:5 dilution.

Add 2ul of 3.2uM primer to each well and cover with a foil seal. 

Label the seal with POC COTS RAPID Putnam 2024XXXX HP001 - HP080



